src/Components/Header.js

```diff
@@ -0,0 +1,18 @@
+import './../styles.css'
+import HeaderRightList from './HeaderRightList'
+import HeaderLeftList from './HeaderLeftList'
+
+function Header() {
+    return (
+        <header>
+            <nav className="left-menu">
+                <HeaderLeftList />
+            </nav>

```

Comment:
There's nothing wrong with this but here's a question to ask of your code:

Is the `<nav className="left-menu">` element the responsibility of the `Header` component, or should it be the responsibility of the `HeaderLeftList` component?

Reply:
I got your point. It would be more efficient to have the navs tags inside the HeaderLeftList and HeaderRightList components.